### DISCUSSION TITLE
有没有什么办法可以构建一个支持原生win11的标题栏

### DISCUSSION THREAD

**Discussion by Ariacha** [hooke007/mpv_PlayKit#307] [libmpv]
好绝望啊，好不容易遇到一款那么nb的播放器但就是这种小细节我却烦恼好久，选择默认标题栏感觉不美观简陋，但是去除windows标题栏装饰之后虽然外观无敌，但是好多原生的功能都无法实现，比如拖拽什么的，要是有什么方法可以让windows自带标题栏全透明然后再自行绘制新的标题栏就好了

**Comment by hooke007** [comment]
这种需求应该向前端提。
考虑到你发的是libmpv板块，但是也没指明是哪个前端。
